my-project/src/LanguageBox.jsx

```diff
@@ -0,0 +1,108 @@
+import { useState, useEffect, useContext } from 'react';
```

Comment:
Try to remove unnecessary imports as they increase resource usage increase the load times.

Reply:
ok